targets/TARGET_STM/watchdog_api.c

```diff
@@ -34,12 +35,12 @@
 // and Watchdog_counter_reload_value bits (RL) of Reload_register (IWDG_RLR)
 // to a timeout value [ms].
 #define PR_RL2UINT64_TIMEOUT_MS(PR_BITS, RL_BITS) \
-	((PR2PRESCALER_DIV(PR_BITS)) * (RL_BITS) * 1000ULL / (LSI_VALUE))
+    ((PR2PRESCALER_DIV(PR_BITS)) * (RL_BITS) * 1000ULL / (LSI_VALUE))
 
 // Convert Prescaler_divider bits (PR) of Prescaler_register (IWDG_PR) and a timeout value [ms]
 // to Watchdog_counter_reload_value bits (RL) of Reload_register (IWDG_RLR)
 #define PR_TIMEOUT_MS2RL(PR_BITS, TIMEOUT_MS) \
-	((TIMEOUT_MS) * (LSI_VALUE) / (PR2PRESCALER_DIV(PR_BITS)) / 1000UL)
+    ((uint32_t)ceil((float)(TIMEOUT_MS) * (LSI_VALUE) / (PR2PRESCALER_DIV(PR_BITS)) / 1000UL))
```

Comment:
Avoid using floating point if possible. The code below will give same values as `ceil()`.
```suggestion
    (((TIMEOUT_MS) * (LSI_VALUE) / (PR2PRESCALER_DIV(PR_BITS)) + 999UL) / 1000UL)
```